src/Components/ArtworkFilter/ArtworkFilters/__tests__/AvailabilityFilter.jest.tsx

```diff
@@ -4,16 +4,14 @@ import {
   createArtworkFilterTestRenderer,
   currentArtworkFilterContext,
 } from "Components/ArtworkFilter/ArtworkFilters/__tests__/Utils"
-import { ArtworkFilterContextProps } from "Components/ArtworkFilter/ArtworkFilterContext"
+import { useArtworkFilterContext } from "Components/ArtworkFilter/ArtworkFilterContext"
 import { AvailabilityFilter } from "Components/ArtworkFilter/ArtworkFilters/AvailabilityFilter"
+import { useEffect } from "react"
+import { __internal__useMatchMedia } from "Utils/Hooks/useMatchMedia"
 
-const artworkFilterContext: Partial<ArtworkFilterContextProps> = {
-  filters: {
-    forSale: undefined,
-  },
-}
+jest.mock("Utils/Hooks/useMatchMedia")
 
-const render = createArtworkFilterTestRenderer(artworkFilterContext)
+const render = createArtworkFilterTestRenderer()
```

Comment:
This call to create the custom test renderer was simplified — a little less explicit but more like the actual implementation. This is so that the filter counts can be calculated correctly in the test.